### ISSUE TITLE
getBase64() misses argument in documentation

### ISSUE THREAD

**Issue by tibo3000** [terrylinla/react-native-sketch-canvas#83]
It seems that it is:
```js
getBase64(imageType, transparent, includeImage, includeText, cropToImageSize, callback)
```
not
```js
getBase64(imageType, transparent, includeImage, cropToImageSize, callback)
```

The example in `example/App.js` is wrong too :)

**Comment by carlosSantillan** [comment]
example/app.js will crash when attempting using this method, error thrown **Cannot have a non-function arg after a function arg.** 
Value for param "cropToImageSize" is missing,

**Comment by GantMan** [comment]
@terrylinla - please update docs, I also had this issue.  I'm happy to PR the change if needed.

**Comment by jramalho** [comment]
So... anyone know how to use it?

**Comment by GantMan** [comment]
Yes, just use the first set of parameters in that order, not the one identified in the docs.  

I got it working like so:  https://github.com/GantMan/ReactNative_MNIST/blob/2d76cc568ba5ca58b2b742eca0aaec5e49521695/App.js#L31-L41

**Comment by hamdidamar** [comment]
yes, includeText parameter should be false and document need update :) thx
